app/javascript/dashboard/routes/dashboard/settings/inbox/FinishSetup.vue

```diff
@@ -21,6 +21,14 @@
           >
           </woot-code>
         </div>
+        <div class="medium-6 small-offset-3">
+          <woot-code
+            v-if="isAGupshupInbox"
+            lang="html"
+            :script="gupshupCallbackURL"
```

Comment:
I'm not sure if I understand this correct. I could find the `gupshupCallbackURL` method or a computed property

Reply:
My bad, it is available on the configMixin.